src/Constants.py

```diff
@@ -38,8 +38,8 @@
 
 # Public RPC
 PUBLIC_NODE_URL = {
-    "MAINNET": "https://rpc.tzkt.io/mainnet",
-    CURRENT_TESTNET: "https://rpc.tzkt.io/{}".format(CURRENT_TESTNET.lower()),
+    "MAINNET": "https://mainnet-tezos.giganode.io",
```

Comment:
Giganode is not available anymore!

Reply:
Since when? Or will it become deprecated in the near future?

Reply:
https://twitter.com/GigaNode/status/1570424065613271040?t=iy7EkkfjnPiUFUS3PbOsVA&s=19

Reply:
Thanks for the heads up :D. What would be the alternative then? https://rpc.tzkt.io/mainnet ?

Reply:
or https://mainnet.smartpy.io pick one you like and that works in your test